### DISCUSSION TITLE
Can we set code block width and position?

### DISCUSSION THREAD

**Discussion by drriguz** [quarto-dev/quarto-cli#4817] [Q&A] [answered]
Hi,
I'm rendering code blocks along with a list（in docx format）：
![image](https://user-images.githubusercontent.com/6427851/224863437-23fbb1c6-1961-45c5-b466-7f7609aa1c08.png)
The code block is always full with and not aligned with the list. What I wanted is like this:
![image](https://user-images.githubusercontent.com/6427851/224864100-85eb4128-1f97-43bd-894b-08e3503fe0d5.png)

How can we achive that? Here's my code:

<pre>
## Install python

If you are using Ubuntu,
you can use:

1. First, install python through package manager:

``` {.bash filename="Ubuntu"}
$ sudo apt update
$ sudo apt install python
```
2. Then, verify that python is installed correctly:

``` {.bash filename="Ubuntu"}
$ python -v
```
</pre>

Thanks!

**Comment by drriguz** [comment, marked as answer]
Aha, interesting, just add some indentation to the code block and it works!

<pre>
1. First, install python through package manager:

  ``` {.bash filename="Ubuntu"}
  $ sudo apt update
  $ sudo apt install python
  ```
</pre>

**Reply by cscheid** [reply]
That happens because now your code cell is a part of the list item. In your previous code, there were _two separate lists_. Markdown is weird in that you don't need to actually number your lists consistently (the feature is there to make editing easier, but it's very weird). Try this:

```
1. Item 1
1. Item 2
1. Item 3
```

and then this:

```
1. Item 1
2. Item 2
Now some text in a new paragraph
3. Item 1
4. Item 2
```

**Reply by drriguz** [reply]
Yeah, that makes sense. It works because it becomes part of the list, not because the identation itself. Thanks for pointing this out!
